examples/polypharmacy_gripnet/README.md

```diff
@@ -17,18 +17,20 @@ The original implementation of GripNet is [here](https://github.com/NYXFLOWER/Gr
 We train GripNet on three datasets constructed by [[2]](https://academic.oup.com/bioinformatics/article/34/13/i457/5045770?login=false).
 The datasets are downloaded from the [Bio-SNAP](http://snap.stanford.edu/biodata/) dataset collection. The integration of these datasets can be regarded as a knowledge graph, which contains the relations among proteins (P) and drugs (D) labeled with the P-P association, the P-D association and 1,317 side effects. The data statistics are shown below.
 
-| Dataset           | Nodes            | Edges   | #Unique Edge Labels |
-| ----------------- | ---------------- | ------- | ------------------- |
-| PP-Decagon        | 19,081(P)        | 715,612 | 1                   |
-| ChG-TargetDecagon | 3,648(P), 284(D) | 18,690  | 1                   |
-| ChChSe-Decagon    | 645(D)           | 63,473  | 1,317               |
+| Dataset           | Nodes            | Edges      | #Unique Edge Labels |
+| ----------------- | ---------------- | ---------- | ------------------- |
+| PP-Decagon        | 19,081(P)        | 715,612    | 1                   |
+| ChG-TargetDecagon | 3,648(P), 284(D) | 18,690     | 1                   |
+| ChChSe-Decagon    | 645(D)           | 4,649,441  | 1,317               |
 
 ## 4. Usage
 - Dataset download: [[here](https://github.com/pykale/data/tree/main/graphs)]
 
 - Algorithm: GripNet
 
-`python main.py --cfg configs/Drug_MINI-GripNet.yaml`
+```python
+python main.py
+```
 
```

Comment:
Consider to add the clarification on the test set (in Skype message) here so users can know as well.

Reply:
Added a TODO and an in-line comment

Reply:
Where is the added TODO and comment? Can you point to me to the code and line number or give me a link? I cannot find them.

Reply:
Comment in Line 69 of main.py?

Reply:
Previously, I added it to the project's TODOs, https://github.com/pykale/pykale/projects/10, but I can not find it now. I will create a new one there.

Reply:
We need to talk about this issue for clarity.

Reply:
I think this should be clarified in the ReadMe of this example.

Reply:
The following is a record of Hao's card in projection board:

gripnet example data: 1) fix the test set, 2) resplit data to train/valid/test sets, and 3) update the polypharmacy gripnet example. 

PS: The problem with the test set is that some side effect classes do not contain any drug pairs.